app/code/Vladimirl/Chatter/Controller/Submit/Submit.php

```diff
@@ -0,0 +1,77 @@
+<?php
+declare(strict_types=1);
+
+namespace Vladimirl\Chatter\Controller\Submit;
+
+use Magento\Framework\Controller\Result\Json as JsonResult;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\App\Action\Action;
+
+class Submit extends Action implements
+    \Magento\Framework\App\Action\HttpPostActionInterface
+{
+    /**
+     * @var \Vladimirl\Chatter\Model\ChatMessage
+     */
+    private $chatMessageFactory;
+    /**
+     * @var \Magento\Framework\DB\TransactionFactory
+     */
+    private $transactionFactory;
+    /**
+     * @var \Magento\Store\Model\StoreManagerInterface $storeManager
+     */
+    private $storeManager;
+
+    /**
+     * Submit constructor.
+     * @param \Magento\Framework\DB\TransactionFactory $transactionFactory
+     * @param \Vladimirl\Chatter\Model\ChatMessageFactory $chatMessageFactory
+     * @param \Magento\Store\Model\StoreManagerInterface $storeManager
+     * @param \Magento\Framework\App\Action\Context $context
+     */
+    public function __construct(
+        \Magento\Framework\DB\TransactionFactory $transactionFactory,
+        \Vladimirl\Chatter\Model\ChatMessageFactory $chatMessageFactory,
+        \Magento\Store\Model\StoreManagerInterface $storeManager,
+        \Magento\Framework\App\Action\Context $context
+    ) {
+        parent::__construct($context);
+        $this->chatMessageFactory = $chatMessageFactory;
+        $this->transactionFactory = $transactionFactory;
+        $this->storeManager = $storeManager;
+    }
+
+    /**
+     * @return \Magento\Framework\App\ResponseInterface|JsonResult|\Magento\Framework\Controller\ResultInterface
+     */
+    public function execute()
+    {
+        $transaction = $this->transactionFactory->create();
+        try {
+            $chatMessage = $this->chatMessageFactory->create();
+            $request = $this->getRequest()->getParam('text_request');
+            $websiteId = (int)$this->storeManager->getWebsite()->getId();
+            $authorType = 'customer';
+            $autorName = 'anonimus';
+            $chatMessage->setAuthorType($authorType)
+                ->setAuthorId(1)
+                ->setAuthorName($autorName)
+                ->setMessage($request)
+                ->setWebsiteId($websiteId)
+                ->setChatHash($request);
```

Comment:
@Vladimir-L Chat Hash must not be the same as message. Hash is something that identifies the chat between the customer and one or multiple store administrators. Read about hashes and hashing algorithms, choose one for your needs - https://ru.wikipedia.org/wiki/%D0%A5%D0%B5%D1%88-%D1%84%D1%83%D0%BD%D0%BA%D1%86%D0%B8%D1%8F